internal/pkg/format/output.go

```diff
@@ -130,6 +130,28 @@ func inferFields[T any](payload T, columns []string) []Field {
 		return
 	}
 
+	dottedToTmpl := func(dotted string) string {
+		parts := strings.Split(dotted, ".")
+		if len(parts) == 2 { // Input has no dots or is just a dot
```

Comment:
Would this really be len 2 if the input has no dots?

Reply:
The `dotter` value is like .a.b.c or .a, in case of .a, the string array looks like ["", "a"] hence checking the length 2. the comment is wrong, will update it.